UI/css/ledgersmb-common.css

```diff
@@ -159,7 +159,7 @@ a.listheading:visited { color: var(--a-listheading-link-clr); font-weight: bold;
 
 .listrow1,
 table.dynatable tr,
-.listrow0 { color: var(--listrow1-bg-clr); font-size: var(--listrow1-font-size); vertical-align: top;
+.listrow0 { color: var(--listrow1-clr); font-size: var(--listrow1-font-size); vertical-align: top;
```

Comment:
these refer to "listrow1-clr" and "listrow1-font-size" yet the class is named listrow0. Is that correct???

Reply:
According to the naming standard I used, variable-clr is for color and variable-bg-clr for background, so it definetely was an error